packages/lib/src/vite-env.d.ts

```diff
@@ -0,0 +1,10 @@
+interface Window {
```

Comment:
Is this necessary for unit tests?

Reply:
This was necessary to avoid ts issues with accessing the new relic object within the lib directory since it doesn't natively exist on `Window`

Reply:
Ah I see, omitting this broke check-types